pages/jobseekers.js

```diff
@@ -0,0 +1,55 @@
+import styled from "styled-components";
+import Link from "next/link";
+import { v4 as uuidv4 } from "uuid";
```

Comment:
When you move the contents of this file over to the `index.js` of this page, make sure to also move the imports and styled components, etc.